src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Blocks/header-details.html.twig

```diff
@@ -0,0 +1,106 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+<div class="product-header-details">
+{% block header_summary %}
+  {% set productData = productForm.vars.data %}
+  <div class="product-header-summary">
+    <div class="product-field-preview">
+      <div class="product-field-value">
+        {# It is important to use vars.data and not vars.value because value contains the localized value which may
+         # not be written with numeric characters anymore depending on the language but data is the initial float data.
+         #}
+        {{ productData.pricing.retail_price.price_tax_excluded|price_format }}
+      </div>
+      <div class="product-field-label">
+        {{ 'tax excl.'|trans({}, 'Admin.Global') }}
+      </div>
+    </div>
+
+    <div class="product-field-preview">
+      <div class="product-field-value">
+        {{ productData.pricing.retail_price.price_tax_included|price_format }}
+      </div>
+      <div class="product-field-label">
+        {# First find the selected tax rate #}
+        {% set taxRate = 0 %}
+        {% for group_label, choice in productForm.pricing.tax_rules_group_id.vars.choices %}
+          {% if choice.data == productData.pricing.tax_rules_group_id %}
+            {% set taxRate = choice.attr['data-tax-rate'] %}
+          {% endif %}
+        {% endfor %}
+
+        {# If tax rate is defined display it in the label #}
+        {% if taxRate %}
+          {{ 'tax incl. (tax rule: %taxRate%%)'|trans({'%taxRate%': taxRate|round(2)}, 'Admin.Catalog.Feature') }}
+        {% else %}
+          {{ 'tax incl.'|trans({'%taxRate%': taxRate|round(2)}, 'Admin.Catalog.Feature') }}
```

Comment:
Shouldn't we use the precision rather than `2` ?

Reply:
Nope, it's not a price which is displayed here It's purely informational to remind the value of the tax rate so 2 decimals are more than enough IMHO